server/route.go

```diff
@@ -538,6 +540,9 @@ func (s *Server) addRoute(c *client, info *Info) (bool, bool) {
 
 		s.routes[c.cid] = c
 		s.remotes[id] = c
+		c.mu.Lock()
+		c.route.connectURLs = info.ClientConnectURLs
```

Comment:
Should we copy?

Reply:
`info.ClientConnectURLs` is immutable: we get this from the unmarshal of a route INFO. We could make a copy if you think the above somehow could pin the byte array we unmarshal from.